Sources/Datadog/RUM/AutoInstrumentation/Views/UIKitRUMViewsPredicate.swift

```diff
@@ -58,15 +58,20 @@ public struct DefaultUIKitRUMViewsPredicate: UIKitRUMViewsPredicate {
     public init () {}
 
     public func rumView(for viewController: UIViewController) -> RUMView? {
-        let canonicalClassName = viewController.canonicalClassName
-        let isCustomClass = canonicalClassName.contains(".") // custom class contains module prefix
-
-        if isCustomClass {
-            var view = RUMView(name: canonicalClassName)
-            view.path = canonicalClassName
-            return view
-        } else {
+        guard !isUIKit(class: type(of: viewController)) else {
+            // do not track bare UIKit's view controllers (UINavigationController, UITabBarController, ...)
             return nil
         }
+
+        let canonicalClassName = viewController.canonicalClassName
+        var view = RUMView(name: canonicalClassName)
+        view.path = canonicalClassName
+        return view
+    }
+
+    /// If given `class` comes from UIKit framework.
+    private func isUIKit(`class`: AnyClass) -> Bool {
```

Comment:
Nitpick: Put this as an internal extension on `Bundle` in the `UIKitExtensions.swift` file

Reply:
Do you mean sth like `Bundle.isUIKit(class:) -> Bool`? This would read weird, no? This seems to be rather a characteristic of the `AnyClass` than a `Bundle` - but as `AnyClass` is a metatype it cannot be extended.

Reply:
it's not important for me but i was thinking of smth like this:
```
// extension
extension AnyObject {
  var isUIKit: Bool { ... }
}
// calling site
if viewController.isUIKit { return }
else { ... }
```

Reply:
Done 👌 , I added:
```swift
internal extension Bundle {
    var isUIKit: Bool
}
```
extension and bunch of fixtures in tests.